sql-db/narayana-transactions/src/test/java/io/quarkus/ts/transactions/SwaggerUiIT.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.ts.transactions;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+
+@QuarkusScenario
+@DisabledOnNative
+public class SwaggerUiIT {
+
+    @DevModeQuarkusApplication
+    static RestService app = new RestService();
+
+    @Test
+    public void smokeTestSwaggerUi() {
+        given()
+                .when().get("/q/swagger-ui")
+                .then()
+                .statusCode(200)
+                .body(containsString("/openapi"));
```

Comment:
Shouldn't we check at least some content?

Reply:
I think that is just a smokeTest, the opanAPI is up and running, is not intended to cover OpenAPI on transaction API scenario.